lib/convert.js

```diff
@@ -124,17 +148,22 @@ var sdk = require('postman-collection'),
 
       // ---- Collection Structure and scaffolding ----
       let specWrapper = {},
-
-        /**
-         We need a trie because the decision of whether or not a node
-        is a folder or request can only be made once the whole trie is generated
-        */
+        folderObj, folderStrategy;
+      specWrapper.spec = openapi;
+
+      /**
+       We need a trie because the decision of whether or not a node
+      is a folder or request can only be made once the whole trie is generated
+      */
+      folderStrategy = _.get(options, 'folderStrategy', 'path');
```

Comment:
The default options should be encoded like they are in the codegens. In the beginning, we can use _.merge - don't need to worry about defaults later in the code.